script.skin.helper.service/plugin.py

```diff
@@ -46,8 +46,36 @@ def __init__(self):
             elif action == "LAUNCH":
                 xbmcplugin.setResolvedUrl(handle=int(sys.argv[1]), succeeded=False, listitem=xbmcgui.ListItem())
                 path = sys.argv[2].split("&path=")[1]
-                xbmc.executebuiltin("Action(Close)")
                 xbmc.executebuiltin(path)
+            elif action == "FOCUSANDCLICK":
+                #used as workaround to display local media in extendedinfo actorinfo
```

Comment:
@phil65 now that you're reviewing the code. This is some ugly workaround I used to get a "in library" container added to extendedinfo.
This is something I'd like to remove from here and add it to extendedinfo. Just another container that will list the local library items for an actor.

Is this something you'd like me to PR to you or you want to implement yourself ?